### ISSUE TITLE
Is there a way to style the data?

### ISSUE THREAD

**Issue by 416serg** [ongoworks/meteor-pdf#9]
Because my template is an invoice that I want saved as PDF, how do I manipulate the styling of `data` instead of each individual piece of information getting outputted on a new line of the document

**Comment by keeev** [comment]
Have you figured it out so far? Trying to do the same.

**Comment by aaronjudd** [comment]
we've pretty much stopped using jspdf because html rendering / styling was difficult.  (they do say it's early stage). however this package is really just a wrapper, you'll get the best info at http://parall.ax/products/jspdf

**Comment by keeev** [comment]
Okay thanks Aaron! :) 
Maybe we/I can figure out some solution with Bootstrap like [here](http://www.sitepoint.com/create-invoice-twitter-bootstrap-part-1/) which was my initial way of doing it but it didn't work so far. 

We'll see.

**Comment by aaronjudd** [comment]
I found if I just stuck with basic html I was able to get some workable layouts, but figuring out what works and what doesn't is a bit tedious.

**Comment by Ajaxsoap** [comment]
Yeah, I literally used different kinds of layout and it didn't worked, even a simple table layout rendered looking weird.

```
<table>
  <tr>
    <td>cell 1</td>
    <td>cell 2</td>
  </tr>
  <tr>
    <td>cell 1</td>
    <td>cell 2</td>
  </tr>
</table>
```
looks like this
![jspdf](https://cloud.githubusercontent.com/assets/6571832/9289560/a357d918-43a7-11e5-9bee-6a6d58afdc69.png)

It would be nice if this issues will fix soon or somehow will enhance for the benefit of the community.
